### PR TITLE
Clean up err codes

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Dylan-DPC

### CODE REVIEW

File: src/librustc_error_codes/error_codes/E0191.md [hunk 1]
```diff
@@ -10,8 +11,9 @@ type Foo = Trait; // error: the value of the associated type `Bar` (from
                   //        the trait `Trait`) must be specified
 ```
 
-Please verify you specified all associated types of the trait and that you
-used the right trait. Example:
+Trait objects need to have all associated types specified. Please verify you
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0191.md]:
```suggestion
Trait objects need to have all associated types specified. Please verify that 
```

File: src/librustc_error_codes/error_codes/E0191.md [hunk 2]
```diff
@@ -10,8 +11,9 @@ type Foo = Trait; // error: the value of the associated type `Bar` (from
...
+specified all associated types of the trait and that you used the right trait.
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0191.md]:
```suggestion
all associated types of the trait were specified and the correct trait was used.
```

File: src/librustc_error_codes/error_codes/E0191.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
-Trait objects need to have all associated types specified. Erroneous code
-example:
+A trait object's associated type wasn't specified.
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0191.md]:
```suggestion
An associated type wasn't specified for a trait object
```

**Comment by GuillaumeGomez** [comment]
Updated.

**Pull request merged**